SYCL/AtomicRef/add.h

```diff
@@ -159,12 +163,34 @@ void add_post_inc_test(queue q, size_t N) {
 
 template <typename T, typename Difference = T>
 void add_test(queue q, size_t N) {
```

Comment:
Minor nit that could be applicable for other tests as well - but feel free to decide on whether this cosmetic change is necessary.
```suggestion
void add_global_test(queue q, size_t N) {
```

Reply:
I believe it is a temporary solution just to let the tests pass with CUDA backend since the backend has had no support for the generic address space yet. When the support for generic address space is added, all the `*_generic.cpp` files will be removed and so, the name `*_test` without any markers (global, generic, etc.) will the only for use.